### PR TITLE
Updating rubygem-activerecord-nulldb-adapter to 0.7.0

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec [hunk 1]
```diff
@@ -5,8 +5,8 @@
 %global gem_name activerecord-nulldb-adapter
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.0
-Release: 2%{?dist}
+Version: 0.7.0
```
Comment (evgeni) [inline packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec:8]: why not 0.8.0?

Comment (Odilhao) [reply packages/foreman/rubygem-activerecord-nulldb-adapter/rubygem-activerecord-nulldb-adapter.spec]: On github the latest version is 0.7.0, adding 0.8.0 now

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
```
- %exclude %{gem_instdir}/.travis.yml
+ %exclude %{gem_instdir}/.github
```

**Comment by ehelms** [comment]
Great minds think alike -- https://github.com/theforeman/foreman-packaging/pull/7971

**Pull request merged**